tests/ignite/metrics/test_average_precision.py

```diff
@@ -34,8 +34,8 @@ def test_no_update():
         ap.compute()
 
 
-def test_input_types():
-    ap = AveragePrecision()
+def test_input_types(available_device):
```

Comment:
Same here: No need to add available_device fixture to the tests checking if an error is raised